.github/workflows/implant-tests.yml

```diff
@@ -1,35 +1,18 @@
-name: Tests
+name: Implant Tests
 
 on:
   workflow_dispatch: ~
   push:
     branches: [ main ]
   pull_request:
     branches: [ main ]
+    paths:
+      - implants/**
```

Comment:
I think some stuff in bin needs to be included.
`create_file_dll` and `reflective_loader` comes to mind.
Also I'd double check how the grpc defs get copied from tavern ---> implants.